src/vm_pmu.rs

```diff
@@ -10,6 +10,9 @@ use sbi_rs::{
     PmuEventType, Result as SbiResult,
 };
 
+#[cfg(test)]
+use test_system::TestResult;
```

Comment:
can we keep all the test related stuff in each module in a `test` sub-module like one does with ordinary rust unit tests? that would reduce the conditional compilation and keep all the test stuff in one place.

Reply:
done